### PR TITLE
Implemented Coordinator node

### LINKED ISSUE #19: Backend module to submit filename and submit time to the coordinator for the new scan
Master module to listen to MongoDB updates

### LINKED ISSUE #22: Run the ClamAV scan on each file and update the database for status
Redis Queue job

### LINKED ISSUE #26: Develop a module to listen to ClamAV updates and update the databse for percentage completion status

### PR DESCRIPTION
**Pull request by maanas-talwar**
Closes #19, closes #22, closes #26 

runningScan

@kmehant @ravindudsilva @1209973 Please review this pull request and share your feedback on the same.
Thanks

### CODE REVIEW

File: Worker/scanJob.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+# RQ job
+def scan(filePath):
+    id = filePath.split("/")[-2]
+    queued = list(queuedScans.find({"_id": id}))
+    if(len(queued) != 0):
+        runningScans.insert_one(queued[0])
+        queuedScans.delete_one({"_id": id})
+    result = cd.scan(filePath)
+    
+    print(result)
```
Comment (1209973) [inline Worker/scanJob.py:29]: @maanas-talwar Do we need to print the results ?

**Review by 1209973 — CHANGES_REQUESTED** [review]

File: Worker/scanJob.py (hunk 1 above)
Comment (maanas-talwar) [reply Worker/scanJob.py:29]: @1209973 I'm printing the results to the console for now. I will be working on saving the results to a file next.

Comment (maanas-talwar) [reply Worker/scanJob.py:29]: Sounds good?

Comment (1209973) [reply Worker/scanJob.py:29]: yeap

**Review by 1209973 — APPROVED** [review]

**Pull request merged**